### DISCUSSION TITLE
help: Can anyone demonstrate a submit.zip file

### DISCUSSION THREAD

**Discussion by CuiZhengliang** [PaddlePaddle/PaddleSpatial#81] [BaiduKDDCup2022] [answered]

**Comment by CuiZhengliang** [comment, marked as answer]
maybe i get it ...

**Reply by moonquakes66** [reply]
Could you tell me how to do?

**Comment by ramber1836** [comment]
Please provide your team name if the error keeps showing.
The administrator will reply to you in couple of days (due to the hard work
load) by checking the backend log.

Thanks.

Regards,
Xinjiang Lu
